examples/cs/grpc/generics/grpc_generics.csproj

```diff
@@ -18,15 +18,9 @@
     <!-- End Resharper Workaround -->
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Attributes">
-      <HintPath>$(BOND_BINARY_PATH)\net45\Bond.Attributes.dll</HintPath>
-    </Reference>
-    <Reference Include="Bond">
-      <HintPath>$(BOND_BINARY_PATH)\net45\Bond.dll</HintPath>
-    </Reference>
-    <Reference Include="Bond.IO">
-      <HintPath>$(BOND_BINARY_PATH)\net45\Bond.IO.dll</HintPath>
-    </Reference>
+    <ProjectReference Include="..\..\..\..\cs\src\attributes\Attributes.csproj" />
+    <ProjectReference Include="..\..\..\..\cs\src\core\Bond.csproj" />
+    <ProjectReference Include="..\..\..\..\cs\src\io\IO.csproj" />
```

Comment:
Minor: merge these with the other `ProjectReference` items, below.
Similarly for the other gRPC examples.

Reply:
I didn't merge them in case they were being parsed and displayed in VS in different groups. I did all the changes on unix so just guessing.

Reply:
Fixed in https://github.com/microsoft/bond/pull/1074/commits/b5520d75e89764b3e09cc4eeb52c9bba2a8b1005

Reply:
awesome